packages/fxa-content-server/tests/functional/verification_reminders.js

```diff
@@ -46,7 +46,7 @@ registerSuite('verification reminders', {
         // The first email is a code, we need to wait for verification
         // reminders which contain a link.
         .then(sendVerificationReminders())
-        .then(getVerificationLink(email, 1))
```

Comment:
Whaaa? Why are these affected?  😕

Reply:
Let's not merge until we figure out what's going on here.

Reply:
Yeah, I'm pretty confused about those. I won't merge until I know why that's happening.

Reply:
I figured out what the issue was btw, just sorting out some type declaration issues now. Should be updated soon-ish.